python/tvm/relay/op/dyn/_transform.py

```diff
@@ -45,12 +45,15 @@ def _reshape_shape_func_input_data(data_shape, newshape, ndim):
     for i in const_range(len(newshape)):
         if skip > 0:
             skip -= 1
-        elif newshape[i] > 0:
+        elif newshape[i] >= 0:
             out[dst_idx] = int64(newshape[i])
             src_idx += 1
             dst_idx += 1
         elif newshape[i] == 0:
```

Comment:
won't this codepath not be hit now? this seems a little sus (since above is >= 0 now). That seems incorrect to cover the case where allowzero != 0

Reply:
Isn't > 0 the comparison we want?

Reply:
Yes you're right, I think this is a mistake that got left over during debugging.